aibolit/patterns/string_concat/string_concat.py

```diff
@@ -33,18 +33,20 @@ def value(self, filename: str) -> List[int]:
         ast = AST.build_from_javalang(build_ast(filename))
         for node in ast.get_proxy_nodes(ASTNodeType.BINARY_OPERATION):
             if node.operator == '+':
+                # by this check we want to filter nodes and consider only
+                # [Literal, MemberReference, MethodInvocation, This]
```

Comment:
How did you get this list?
When I check node types for `value` field, I get following:
 - ASSERT_STATEMENT
 - ASSIGNMENT
 - ELEMENT_VALUE_PAIR
 - LITERAL
 - TRY_RESOURCE

By the way, can you explain, how other than LITERAL nodes meet this pattern?